src/main/scala/rocket/CSR.scala

```diff
@@ -214,6 +212,7 @@ class CSRFile(perfEventSets: EventSets = new EventSets(Seq()))(implicit p: Param
   reset_dcsr.xdebugver := 1
   reset_dcsr.prv := PRV.M
   val reg_dcsr = Reg(init=reset_dcsr)
+  val reg_debugint = Reg(init=Bool(false), next=io.interrupts.debug)
```

Comment:
I was just taking it from its previous functionality where reset_dcsr was 0. But I agree it shouldn't be reset.